src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.ts

```diff
@@ -61,14 +63,19 @@ export class CephfsSubvolumeListComponent extends CdForm implements OnInit, OnCh
   selectedName: string = '';
 
   subVolumes$: Observable<CephfsSubvolume[]>;
+  subVolumeGroups$: Observable<CephfsSubvolumeGroup[]>;
   subject = new ReplaySubject<CephfsSubvolume[]>();
 
+  subVolumeGroupName: string = '';
```

Comment:
more meaningful name would be
```suggestion
  activeGroupName: string = '';
```